src/java.desktop/share/classes/javax/swing/ImageIcon.java

```diff
@@ -367,10 +370,14 @@ public Image getImage() {
 
     /**
      * Sets the image displayed by this icon.
+     * Setting null image will not render any image icon.
```

Comment:
```suggestion
     * Setting a {@code null} image will not render any image icon.
```

Will it sound clearer if the text stated, _“Setting a {@code null} image will remove the icon?”_